<?xml version="1.0"?>
<launch>
  <node name="circle_test_source" pkg="circle_test_source" type="talker.py">
    <param name="radius"                     value="1.0" />
    <param name="omega_max"                  value="1.0" />
    <param name="sampling_freq"              value="200" />
    <param name="envelope_freq"              value="0.1591549431" />
  </node>
  <node name="ros_melodic_ronin" pkg="ros_melodic_ronin" type="ronin.py">
    <param name="step_size"                  value="10" />
    <param name="window_size"                value="200" />
    <param name="cpu"                        value="True" />
    <param name="model_path"                 value="/home/balazs/munka/nowtech/repos/nowtechnologies/ronin/models/ronin_resnet/checkpoint_gsn_latest.pt" />
  </node>
</launch>
